net-im/bitlbee/bitlbee-3.5.1.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="IRC to IM gateway that support multiple IM protocols"
+DESCRIPTION="BitlBee brings IM (instant messaging) to IRC clients. It's a \
+great solution for people who have an IRC client running all the time and \
+don't want to run an additional MSN/AIM/whatever client.
+BitlBee currently supports the following IM networks/protocols: XMPP/Jabber \
+(including Google Talk and Hipchat), MSN/Skype, Yahoo! Messenger, AIM and \
+ICQ, the Twitter microblogging network (plus all other Twitter API \
+compatible services like status.net)"
+HOMEPAGE="https://www.bitlbee.org/"
+COPYRIGHT="2002-2013 Wilmer van der Gaast and others"
```

Comment:
```diff
-COPYRIGHT="2002-2013 Wilmer van der Gaast and others"
+COPYRIGHT="2002-2017 Wilmer van der Gaast and others"
```
or maybe
```
COPYRIGHT="2002-2017 Wilmer van der Gaast and others
	2014-2017 \"dequis\""
```
Anyway, it would be nice to show a `2017`.